ui/src/app/apps/apps.service.ts

```diff
@@ -50,38 +48,38 @@ export class AppsService {
 
   getAppInfo(appType: ApplicationType, appName: string): Observable<DetailedAppRegistration> {
     console.log(this.appRegistrations);
-    let headers = new Headers({ 'Content-Type': 'application/json' });
-    let options = new RequestOptions({ headers: headers });
+    const headers = new Headers({ 'Content-Type': 'application/json' });
```

Comment:
Should we consider static readonly at class level instead of redeclaring `const headers`?